npm-shrinkwrap.json

```diff
@@ -16143,15 +17636,38 @@
       }
     },
     "reakit": {
-      "version": "1.1.2",
-      "resolved": "https://registry.npmjs.org/reakit/-/reakit-1.1.2.tgz",
-      "integrity": "sha512-UJEeWJ+OwtBBY6gpOkbc2rflASID+t+B6GlqCmqwO4eD0YIGJwmF2O5U7HzV5igtyNBGPyKMhpodm2Yhx6UfHw==",
+      "version": "1.3.5",
+      "resolved": "https://registry.npmjs.org/reakit/-/reakit-1.3.5.tgz",
+      "integrity": "sha512-Luv1RPBFlWhRG32Ysjd86KC+vLoz5da3X0O8rqClaNEv259nWmnw5fG6BIUSYJwTG6PPxTidPlS+9bS6nLstfA==",
       "requires": {
-        "@popperjs/core": "^2.4.4",
-        "body-scroll-lock": "^3.0.2",
-        "reakit-system": "^0.13.1",
-        "reakit-utils": "^0.13.1",
-        "reakit-warning": "^0.4.1"
+        "@popperjs/core": "^2.5.4",
+        "body-scroll-lock": "^3.1.5",
+        "reakit-system": "^0.15.1",
+        "reakit-utils": "^0.15.1",
+        "reakit-warning": "^0.6.1"
+      },
+      "dependencies": {
+        "reakit-system": {
+          "version": "0.15.1",
+          "resolved": "https://registry.npmjs.org/reakit-system/-/reakit-system-0.15.1.tgz",
+          "integrity": "sha512-PkqfAyEohtcEu/gUvKriCv42NywDtUgvocEN3147BI45dOFAB89nrT7wRIbIcKJiUT598F+JlPXAZZVLWhc1Kg==",
+          "requires": {
+            "reakit-utils": "^0.15.1"
+          }
+        },
+        "reakit-utils": {
+          "version": "0.15.1",
+          "resolved": "https://registry.npmjs.org/reakit-utils/-/reakit-utils-0.15.1.tgz",
+          "integrity": "sha512-6cZgKGvOkAMQgkwU9jdYbHfkuIN1Pr+vwcB19plLvcTfVN0Or10JhIuj9X+JaPZyI7ydqTDFaKNdUcDP69o/+Q=="
+        },
+        "reakit-warning": {
+          "version": "0.6.1",
+          "resolved": "https://registry.npmjs.org/reakit-warning/-/reakit-warning-0.6.1.tgz",
+          "integrity": "sha512-poFUV0EyxB+CcV9uTNBAFmcgsnR2DzAbOTkld4Ul+QOKSeEHZB3b3+MoZQgcYHmbvG19Na1uWaM7ES+/Eyr8tQ==",
+          "requires": {
+            "reakit-utils": "^0.15.1"
```

Comment:
This was 0.14.4 by default but it was causing a conflict with the version required by @wordpress/components, so I manually bumped it to 0.15.1. It looks some level of instability is expected around the version updates with reakit-utils: "[This is experimental and may have breaking changes in minor versions.](https://www.npmjs.com/package/reakit-utils)"